LondonBritishLibrary/orient/BLorient810.xml

```diff
@@ -30,20 +31,88 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 385</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
+                        <msItem xml:id="ms_i1">
+                          <locus from="4r"></locus>
+                          <title type="incomplete" ref="LIT1011Abusak"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <note>Chapters 47, 49, 51, 53, 58 and 59 are missing.</note>
+
+                          <msItem xml:id="ms_i1.1">
+                            <locus from="4r"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Index"></title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.2">
+                            <locus from="7r"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter1"></title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.3">
+                            <locus from="91v"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter48"></title>
+                            <textLang mainLang="gez"></textLang>
+                            <note>Numbered 47.</note>
+                          </msItem>
+                          <msItem xml:id="ms_i1.4">
+                            <locus from="150v"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter50"></title>
+                            <textLang mainLang="gez"></textLang>
+
+                            <msItem xml:id="ms_i1.4.1">
+                              <title type="complete" xml:lang="en">The list of the patriarchs
+                              of Alexandria</title>
+                              <textLang mainLang="gez"></textLang>
+                              <note>The list ends with <persName ref="PRS6752Mark"></persName>.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.4.2">
+                              <locus target="#178r"></locus>
+                              <title type="complete" xml:lang="en">The Seven Wonders of the World</title>
+                              <textLang mainLang="gez"></textLang>
+                            </msItem>
+                          </msItem>
+                          <msItem xml:id="ms_i1.5">
+                            <locus from="178r"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter52"></title>
```

Comment:
I guess when Wright says "Chapter 52" etc. he means that there is the corresponding label in the text, so am wondering if it should be put inside of the `<title>` tag or as a `<rubric>`